api4/apitestlib.go

```diff
@@ -93,6 +93,8 @@ func setupTestHelper(dbStore store.Store, searchEngine *searchengine.Broker, ent
 	options = append(options, app.ConfigStore(memoryStore))
 	options = append(options, app.StoreOverride(dbStore))
 
+	mlog.DisableZap()
```

Comment:
Is it possible to do the disabling inside the TestMain functions? Since this is purely for tests, that might be a better place for it.

Reply:
This seemed to modify a higher level logic than the usual app-level setup performed in `setupTestHelper`. That's why I wanted to move it inside `TestMain`. It's not exactly a lot of deduplication either; it is already duplicated multiple times in api4, app, web and other packages.

But it's temporary code at the end of day. So if you feel this is okay, I don't mind.

Reply:
Ok, I moved all the DisableZap calls to TestMain.